### PR TITLE
✨ Added My6sense adapter to amp project

### PR DESCRIPTION
**Pull request by my6sense**

### CODE REVIEW

**Comment by calebcordry** [comment]
Hi @my6sense thanks for the contribution! Let me know if you have trouble sorting out the CLA process.

**Comment by my6sense** [comment]
Hi @calebcordry, thanks for the quick response.
1. What we need to do to fix the CLA issue ? 
2. Is the owners-check and percy/amphtml issues relevant to us ? we need to handle this ?
Thanks a lot!

**Comment by calebcordry** [comment]
@my6sense looks like the commit author is different than the PR author, I am guessing the commit is from your work email, and I don't see an email associated with this account that opened the PR. You might need to add an email to this `my6sense` account and ensure the commit is from the same author. There should be some more info from the googlebot above.

Don't worry about owners check, that will go away when I approve, and percy is probably a flake.

File: ads/my6sense.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/my6sense.md]: nit: 2020

File: examples/ads.amp.html [hunk 2]
```diff
@@ -1536,6 +1537,16 @@ <h2>mox</h2>
       data-h="405">
   </amp-ad>
 
+  <h2>My6sense</h2>
```
Comment (calebcordry) [inline examples/ads.amp.html]: I did not see an example creative load here, would it be possible to return a sample ad? It helps us catch regressions etc.

File: ads/my6sense.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (my6sense) [reply ads/my6sense.js]: fixed!

File: ads/my6sense.md (hunk 1 above)
Comment (my6sense) [reply ads/my6sense.md]: fixed!

File: examples/ads.amp.html (hunk 2 above)
Comment (my6sense) [reply examples/ads.amp.html]: Hi @calebcordry, when i'm running this file locally i'm getting our sample ad,
Can you pls tell me if you are getting widget3.js in network tab, if so can you share it with me, tnx!

**Comment by calebcordry** [comment]
Sure! Screenshot of creative:
image

w3 response [here](https://gist.github.com/calebcordry/161d21b21a7f67a4eabf3b8345858c55)

**Comment by my6sense** [comment]
Hi @calebcordry,
I figured out what's the problem, i'm developing on desktop env and the widget key is for desktop widgets.
I changed the widget type to mobile and now it should work.

**Comment by my6sense** [comment]
@calebcordry, thank you for approving!
i've got a small question: when i try to create an AMP page and use our tag i get this error: Unknown ad type "my6sense"​​​.

i'm following the steps as shown here: 
https://amp.dev/documentation/guides-and-tutorials/start/create/basic_markup
its using this file https://cdn.ampproject.org/v0.js as the script and it looks like our "adapter" isn't included there.

what am i doing wrong?
thanks!

**Comment by calebcordry** [comment]
@my6sense Your changes are merged, but have not hit a release yet. They should be in our `experimental` release around Tuesday (which you can opt into) and Prod the following week. You can find more details [here](https://amp.dev/documentation/guides-and-tutorials/learn/spec/release-schedule/)

**Pull request merged**